package.json

```
@@ -84,7 +84,7 @@
     "fast-xml-parser": "^4.5.2",
     "glob": "8.1.0",
     "mini-css-extract-plugin": "^2.7.6",
-    "mocha": "10.2.0",
+    "mocha": "10.8.2",
```

```suggestion
    "mocha": "11.5.0",
```

This [dependabot PR](https://github.com/apache/daffodil-vscode/pull/1278) updates the version to 11.5.0, so is this PR necessary or should we use the dependabot one?